notebooks/02_model/hypertune_spark_deep_dive.ipynb

```diff
@@ -0,0 +1,983 @@
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "model.explainParams()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[1.3818693574087881,\n",
+       " 1.03881169247089,\n",
+       " 1.4149480500849774,\n",
+       " 1.8957753237837804,\n",
+       " 2.707401263212467,\n",
+       " 1.3818693597518183,\n",
+       " 1.022026009040587,\n",
+       " 1.4517896758192947,\n",
+       " 1.9079444576340328,\n",
+       " 2.786421555547063]"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "model.validationMetrics"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To get the best model, just do"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_best_spark = model.bestModel"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3 Hyperparameter tuning with `hyperopt` in sequential mode"
```

Comment:
According to the name in here https://github.com/hyperopt/hyperopt/wiki, Hyperopt is Distributed asynchronous algorithm configuration / hyperparameter optimization. I don't understand why we are using it in `sequential mode`

Reply:
Ah this was not made clear. `hyperopt` does not support distribution on Spark.

Reply:
I changed the section title and added some text to make it more clear.